Makefile

```diff
@@ -3,6 +3,7 @@
 .PHONY : cabal-install-dev cabal-install-prod
 
 LEXER_HS:=Cabal/Distribution/Fields/Lexer.hs
+LEXER_CHS:=Cabal/Distribution/C2Hs/Lexer.hs
```

Comment:
We still need to commit in `Lexer.hs` file too. So the people who do not modify `Lexer.x` shouldn't need to be aware of it.

Reply:
Should be done.